guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -9,15 +9,15 @@ To create a file type repository in a directory on the base system where {Projec
 
 .Prerequisites
 ifdef::katello,orcharhino[]
-* You have a server running {EL} 8 registered to your {Project}.
+* You have a server running {EL} 9 or {EL} 8 registered to your {Project}.
 endif::[]
 ifdef::satellite[]
-* You have a server running {EL} 8 registered to your {Project} or the Red{nbsp}Hat CDN.
+* You have a server running {EL} 9 registered to your {Project} or the Red{nbsp}Hat CDN.
```

Comment:
Is there a reason why we allow users to use EL 9 or 8 in upstream but only EL 9 in Satellite?

Reply:
I have a question out to Ian about this.

Reply:
RHEL 8 and RHEL 9 should work in all cases, not just upstream.

Reply:
Actually, I'm chatting with Odilon now about where the pulp_manifest package actually lives, it may not even be in the utils repo any more. Let me confirm first.

Reply:
Okay, so it turns out the repository where pulp_manifest is stored completely changed. Even for Satellite 6.15.

The repository is now the main Satellite one: satellite-6.15-for-rhel-8-x86_64-rpms

For 6.16, it'll be satellite-6.16-for-rhel-8-x86_64-rpms and satellite-6.16-for-rhel-9-x86_64-rpms

For upstream, the pulp_manifest package is available in the pulpcore repository, which will already be enabled on the Katello machine.

Reply:
I verified that the docs are correct for Satellite 6.14. So it's just 6.15+ that need correction.

Reply:
Changed the attribute to reflect the new repository.

Reply:
@bangelic Ian says that it should run on both RHEL 9 and 8. So:
```suggestion
* You have a server running {EL} 9 or {EL} 8 registered to your {Project} or the Red{nbsp}Hat CDN.
```
But this isn't the only change required. You have also provide the step to enable the repos on EL 8 below.